assets/js/view/part/track.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'snabberb/component'
+
+require 'view/part/track_curvilinear_path'
+require 'view/part/track_lawson_path'
+require 'view/part/track_offboard'
+
+module View
+  module Part
+    class Track < Snabberb::Component
+      ROUTE_COLORS = %i[red blue green purple].freeze
+
+      needs :tile
+      needs :region_use
+      needs :routes
+
+      def render
+        return if @tile.exits.empty?
+
+        # each route has an "entry" in this array; each "entry" is an array of
+        # the paths on that route that are also on this tile
+        #
+        # Array<Array<Path>>
+        @routes_paths = @routes.map { |route| route.paths_for(@tile.paths) }
+
+        if !@tile.offboards.empty?
+          track_class = Part::TrackOffboard
+          paths = @tile.paths.select { |p| [p.a, p.b].any?(&:offboard?) }
+        elsif @tile.lawson?
+          track_class = Part::TrackLawsonPath
+          paths = @tile.paths.select { |p| p.edges.size == 1 }
```

Comment:
Apparently not, will take a closer look after I push the other changes I've got so far

Reply:
Figured out why this doesn't work--I considered tiles with one city (e.g., 5 and 6) as lawson tiles because the track paths they use are basically the same as the paths on a tile like 81A--straight from the edge to the center.